tools/tune/dcblock/example_dcblock.m

```diff
@@ -1,30 +1,38 @@
-function example_dcblock();
+function example_dcblock()
 
 % Set the parameters here
-tplg_fn = "../../topology/topology1/m4/dcblock_coef_default.m4" % Control Bytes File
+tplg1_fn = "../../topology/topology1/m4/dcblock_coef_default.m4" % Control Bytes File
```

Comment:
Please include a semicolon in all parameter settings if you do not intend to print the output to the console. from line 4to line 10

Reply:
Since it's consistent I was wondering if it was intentional, and kept as-is, but I can remove. Please chime in if this is not desired.

Reply:
It appears to be perplexing!! A couple of thoughts.


I would not print to the console since they are assigned to a variable, and if we need to look at the variable, and variables must be clearly defined with the intended meaning attached. 


 Can a rename the blob and text files because they contain the same data for `blob3/4` and `alsa3/4`?

Reply:
Not sure I'm following but the ipc4 blob contains a different header, so the files are not copies.

Reply:
```
blob3_fn = "../../ctl/ipc3/dcblock_coef.blob";% Binary file blob
blob4_fn = "../../ctl/ipc4/dcblock_coef.blob";% Binary file blob

```
```
alsa3_fn = "../../ctl/ipc3/dcblock_coef.txt";% CSV file in ALSA format
alsa4_fn = "../../ctl/ipc4/dcblock_coef.txt";% CSV file in ALSA format

```

I mean, aren't `dcblock_coef.blob` and `dcblock_coef.txt` the same file?

Reply:
It's same data but different format. The binary format is for UCM, sof-ctl can use both binary and ascii decimal. I like myself the ascii decimal to quickly visually check the numbers data.

Reply:
Thanks for the explanation!